### PR TITLE
Update cses-1631.mdx - Java Added

### PR DESCRIPTION
**Pull request by Nithya-02**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-1631.mdx [hunk 1]
```diff
@@ -47,6 +47,39 @@ int main() {
 }
 ```
 </CPPSection>
+<JavaSection>
+
+```java
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+class mxsm
```
Comment (envyaims) [inline solutions/cses-1631.mdx:56]:
```suggestion
public class Main
```

File: solutions/cses-1631.mdx [hunk 2]
```diff
@@ -47,6 +47,39 @@ int main() {
...
+{
+	public static void main (String[] args) throws java.lang.Exception
+	{
+		Scanner sc = new Scanner(System.in);
```
Comment (envyaims) [inline solutions/cses-1631.mdx:60]: use Kattio or buffered reader. This is required in code conventions

File: solutions/cses-1631.mdx [hunk 3]
```diff
@@ -47,6 +47,39 @@ int main() {
...
+		int n = sc.nextInt();
+		long arr[] = new long[n];  
+		for(int i=0; i<n; i++)  
+		{  
+			arr[i]=sc.nextLong();  
+		}  
+		//max-element
+		long maxNum = arr[0];
+        for (long j : arr) {
```
Comment (envyaims) [inline solutions/cses-1631.mdx:69]: weird indenting. use tabs like above

File: solutions/cses-1631.mdx [hunk 4]
```diff
@@ -47,6 +47,39 @@ int main() {
...
+            if (j > maxNum)
```
Comment (envyaims) [inline solutions/cses-1631.mdx:70]: use Math.max

File: solutions/cses-1631.mdx [hunk 5]
```diff
@@ -47,6 +47,39 @@ int main() {
...
+                maxNum = j;
+        }
+        //sum-element
+        long sumNum = 0;
+        for (int j = 0; j<n ; j++) {
```
Comment (envyaims) [inline solutions/cses-1631.mdx:75]: keep your spacings consistent...

File: solutions/cses-1631.mdx [hunk 6]
```diff
@@ -47,6 +47,39 @@ int main() {
...
+	public static void main (String[] args) throws java.lang.Exception
+	{
+		Scanner sc = new Scanner(System.in);
+		int n = sc.nextInt();
+		long arr[] = new long[n];  
+		for(int i=0; i<n; i++)  
+		{  
+			arr[i]=sc.nextLong();  
+		}  
+		//max-element
```
Comment (envyaims) [inline solutions/cses-1631.mdx:67]: remove these random comments

File: solutions/cses-1631.mdx [hunk 7]
```diff
@@ -47,6 +47,39 @@ int main() {
...
+		long maxNum = arr[0];
+        for (long j : arr) {
+            if (j > maxNum)
+                maxNum = j;
+        }
+        //sum-element
+        long sumNum = 0;
+        for (int j = 0; j<n ; j++) {
+        	sumNum = sumNum + arr[j];
```
Comment (envyaims) [inline solutions/cses-1631.mdx:76]: use +=

**Review by envyaims — CHANGES_REQUESTED** [review]

File: solutions/cses-1631.mdx [hunk 8]
```diff
@@ -47,6 +47,39 @@ int main() {
...
+{
+	public static void main (String[] args) throws java.lang.Exception
+	{
+		Scanner sc = new Scanner(System.in);
+		int n = sc.nextInt();
+		long arr[] = new long[n];  
+		for(int i=0; i<n; i++)  
+		{  
+			arr[i]=sc.nextLong();  
+		}
```
Comment (Aadit-Ambadkar) [inline solutions/cses-1631.mdx:66]:
```suggestion
		for(int i = 0; i < n; i++)  {  
			arr[i] = sc.nextLong();  
		}  
```

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**